docs/UserGuide.md

```diff
@@ -141,80 +141,122 @@ Format: `clear`
 Adds a supplier to the address book.
 
 Format: `add -s n/NAME p/PHONE e/EMAIL com/COMPANY [t/TAG]…​ [pro/PRODUCT]…​`
-<box type="tip" seamless>
 
-Parameters:
+<box type="details" seamless>
 
-- `n/NAME`: Must be alphanumeric, and must not be blank.
-- `p/PHONE`: Must be numeric, must contain at least 3 digits, and must not be blank.
-- `e/EMAIL`: Must be a valid email address, and must not be blank.
-- `com/COMPANY`: Must be alphanumeric, and must not be blank.
-- `[t/TAG]`: Must be alphanumeric, and must be between 1 and 50 (inclusive) characters long.
-- `[pro/PRODUCT]`: Must be alphanumeric, can include spaces, and must be between 1 and 50 (inclusive) characters long.
+Parameters:
```

Comment:
since we decided to include the prefix as part of the parameter, some instructions are slightly confusing now

for example:
`p/PHONE`: The supplier's phone number. It must be numeric, and contain at least 3 digits.
this above line suggests that the entire parameter must be numeric while our prefix is still p/.
Maybe you can specify that `PHONE` must be numeric instead.

same applies for the other parameters for all other commands as well

Reply:
I agree with pravien here